### PR TITLE
Improved antiscroll rebuild

### PR DESCRIPTION
**Pull request by taras**
- Added resize-container-test route that shows table that's resized when container size changes
- Added test that shows that scrollbar gets broken when resizing a table
- Added skipped test that shows an unnecessary scrollbar when rendering the table will `columnsFillTable=true`

### CODE REVIEW

**Comment by taras** [comment]
The tests fail for 1.13+, we probably should configure the CI to not fail for 1.13+

**Comment by taras** [comment]
This PR is not ready yet, I missed something. I'm working on it now.

File: addon/components/ember-table.js [hunk 1]
```diff
@@ -282,7 +282,17 @@ StyleBindingsMixin, ResizeHandlerMixin, {
...
-    Ember.run.next(this, this.updateLayout);
+    this.scheduleAntiscrollRebuild();
+  },
+
+  scheduleAntiscrollRebuild() {
+    Ember.run.scheduleOnce('afterRender', this, this.rebuildAntiscroll);
+  },
+
+  rebuildAntiscroll() {
+    if (this._state !== 'inDOM'){ return; }
```
Comment (azirbel) [inline addon/components/ember-table.js]: newlines:
```
if (this._state !== 'inDOM') {
  return;
}
```

**Comment by taras** [comment]
Made the change, do you want me to rebase it?

**Comment by azirbel** [comment]
No need, I can do it. Just want to test it against our app today and see if I can find any problems.

**Comment by billy-addepar** [comment]
This version of Ember table is no longer supported. If you want to continue discussion, you can open the issue on [Ember Table Legacy](https://github.com/Addepar/ember-table-legacy)

**Pull request closed without merging**